### PR TITLE
GEODE-985: rebrand GFSH help for Apache Geode

### PR DESCRIPTION
**Pull request by gracemeilen**

### CODE REVIEW

**Comment by davebarnes97** [comment]
Nice work, Grace - thanks for handling it.
One suggested correction: the name "geode" should be given an initial cap ('Geode') in the string EXPORT_OFFLINE_DISK_STORE__HELP.
Thanks for catching some other typos and getting rid of 'vfabric'.
Question: Once this change is in place, will GemFire's version of gfsh help contain lots of Geode references? If so, has PM okayed that?

File: geode-core/src/main/java/com/gemstone/gemfire/management/internal/cli/i18n/CliStrings.java [hunk 1]
```diff
@@ -63,65 +59,67 @@
...
-  public static final String TOPIC_GEMFIRE_HELP = "Help";
-  public static final String TOPIC_GEMFIRE_HELP__DESC = "Provides usage information for gfsh & its commands.";
-  public static final String TOPIC_GEMFIRE_DEBUG_UTIL = "Debug-Utility";
-  public static final String TOPIC_GEMFIRE_DEBUG_UTIL__DESC = "Debugging aids & utilities to use with vFabric GemFire.";
+  public static final String DEFAULT_TOPIC_GEODE = "Geode";
+  public static final String DEFAULT_TOPIC_GEODE__DESC = "Apache Geode is a distributed data management platform providing dynamic scalability, high performance and database-like persistence.";
```
Comment (scmbuildguy) [inline geode-core/src/main/java/com/gemstone/gemfire/management/internal/cli/i18n/CliStrings.java]: When referencing the projecting as 'Apache Geode', I think we should be using 'Apache Geode (incubating)' until Geode is a top-level Apache project.

File: geode-core/src/main/java/com/gemstone/gemfire/management/internal/cli/i18n/CliStrings.java [hunk 2]
```diff
@@ -63,65 +59,67 @@
...
+  public static final String TOPIC_GEODE_REGION = "Region";
+  public static final String TOPIC_GEODE_REGION__DESC = "A region is the core building block of the Apache Geode distributed system. Cached data is organized into regions and all data puts, gets, and querying activities are done against them.";
```
Comment (scmbuildguy) [inline geode-core/src/main/java/com/gemstone/gemfire/management/internal/cli/i18n/CliStrings.java]: When referencing the projecting as 'Apache Geode', I think we should be using 'Apache Geode (incubating)' until Geode is a top-level Apache project.

**Comment by scmbuildguy** [comment]
I did mention in a couple of places, however, in generalI think when referencing the project as 'Apache Geode', I think we should be using 'Apache Geode (incubating)' until Geode is a top-level Apache project.

**Comment by metatype** [comment]
IIRC, the requirement is to mention (incubating) on the first reference.  Here are the branding requirements:

http://incubator.apache.org/guides/branding.html


Anthony

**Pull request closed without merging**